types/react-onsenui/index.d.ts

```diff
@@ -334,21 +348,33 @@ export class Switch extends Component<{
  * Tabs
  */
 
-export class Tab extends Component { }
+export class Tab extends Component<{
+    label?: string,
+    icon?: string,
+}> { }
 
 export class TabActive extends Component { }
 
 export class TabInactive extends Component { }
 
+export interface TabbarRenderTab {
+    content: JSX.Element;
+    tab: JSX.Element;
+}
+
 export class Tabbar extends Component<{
     index?: number,
-    renderTabs?(): any,
+    renderTabs?(): TabbarRenderTab[],
     position?: "bottom" | "top" | "auto",
-    animation: "none" | "slide" | "fade",
+    swipeable?: boolean,
+    ignoreEdgeWidth?: number,
+    animation?: "none" | "slide" | "fade",
     animationOptions?: AnimationOptions,
+    tabBorder?: boolean,
     onPreChange?(): void,
     onPostChange?(): void,
     onReactive?(): void,
+    onSwipe?(): void,
```

Comment:
According to the documentation: "It gets a decimal index and an animationOptions object as arguments."